doc/developer-notes.md

```diff
@@ -481,22 +481,35 @@ $ ./test/functional/test_runner.py --valgrind
 
 ### Compiling for test coverage
 
-LCOV can be used to generate a test coverage report based upon `make check`
+LCOV can be used to generate a test coverage report based upon `ctest`
 execution. LCOV must be installed on your system (e.g. the `lcov` package
 on Debian/Ubuntu).
 
 To enable LCOV report generation during test runs:
 
 ```shell
-./configure --enable-lcov
-make
-make cov
+cmake -B build --preset coverage
```

Comment:
This is different from the examples in the PR description and also skips building the fuzz binary, is that expected?

Reply:
The PR description has been updated.

Reply:
UPD. Added `--ignore-errors mismatch`